Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -464,6 +464,36 @@ public extension UIView {
 		}
 	}
 
+    /// SwifterSwift: Anchors all sides to superview's safe area.
+    ///
+    /// 1. On iOS 11 and latter anchors view's sides to superview's safe area
+    /// 2. On iOS 9-11 anchors leading and trailing to superview,
+    /// top and bottom to viewController's bottomLayoutGuide and topLayoutGuide.
+    ///
+    /// - Parameter viewController: view controller, of which bottomLahoutGuide
+    ///  and topLayoutGuide are used on iOS 9-11
+    @available(iOS 9, *) public func fillToSuperviewSafeArea(viewController: UIViewController) {
+        guard let superview = self.superview else {
```

Comment:
Maybe you should also add a check for `super == viewController.view`? Does that make sense to be a requirement?

Reply:
Not necessary, safe area works even for subviews.